src/ui/public/courier/data_source/build_query/__tests__/build_es_query.js

```diff
@@ -36,8 +36,8 @@ describe('build query', function () {
 
     it('should combine queries and filters from multiple query languages into a single ES bool query', function () {
       const queries = [
-        { query: 'foo:bar', language: 'kuery' },
-        { query: 'bar:baz', language: 'lucene' },
+        { query: 'machine.os:bar', language: 'kuery' },
+        { query: 'extension:baz', language: 'lucene' },
```

Comment:
Because I updated the kuery functions to throw an error if a non-existent field is queried, I had to update these tests to use fields that actually exist in our stub index pattern.